src/lib/viewers/image/ImageControls.tsx

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
 import AnnotationsControls, { Props as AnnotationsControlsProps } from '../controls/annotations';
+import { Props as ColorPickerControlProps } from '../controls/color-picker';
```

Comment:
Can this be exposed via the AnnotationsControlsProps, instead?

Reply:
There are two props that are referenced in ColorPickerControlProps that are not referenced in AnnotationsControlsProps:
`isActive` and `onAnnotationColorClick`. Since AnnotationsControlsProps properties is not a superset of all of the props of ColorPickerControlProps, I don't think we can remove the above line.

Reply:
I'm a little confused. The only prop added in this PR to this file is `annotationColor`, which is passed down to AnnotationControls and so should be exposed in its exported props object. Why is `ColorPickerControlProps` needed?

Reply:
Yes that's correct, I don't need `ColorPickerControlProps`.